docs/list.md

```diff
@@ -299,6 +299,16 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **How to sign up**: No special signup known.
 - **Notes**: Read more at [JavaScript Today](https://blog.javascripttoday.com/blog/hacktoberfest-2023/).
 
+### L
+
+#### **Loft Labs**
+
+- **Swag**: Loft exclusive prizes.
+- **Requirements**:
+  - Contribute to vcluster and devpod project of Loft Labs.
+- **How to sign up**: Join the [loft labs slack channel](https://slack.loft.sh/)
```

Comment:
I don't see this as a requirement in the tweet. 
Was this updated somewhere? Can you add that link too?

Please update the link twitter link below with the actual tweet link or the updated announcement link.

Reply:
Yes they have mentioned in one of the tweet that only these 2 projects would be counted for exclusive prizes [tweet link](https://twitter.com/loft_sh/status/1710331345526800719).
Working on all the other suggestions you have made 👍

Reply:
I am sorry, I meant to ask about the sign up part. The requirement to join their slack channel.

Reply:
Oh yess it's a mistake there. There is no any special sign up is known for loft lab. I'm sorry updating this 👍. And add on I have updated the specific tweet link and added the issues part mentioning issues only for devpod and vcluster.